backend/src/main/java/gov/cdc/usds/simplereport/db/model/Person.java

```diff
@@ -33,8 +35,9 @@
 	private StreetAddress address;
 	@Column
 	private String gender;
-	@Column
-	private String race;
+	@Type(type = "list-array")
+	@Column(columnDefinition = "text ARRAY") // needed by auto-ddl validation, just for the array type
```

Comment:
I'm not 100% sure this is needed now that the underlying bug is fixed—if it is, we should probably say `text[]` rather than `text ARRAY` because that's the type we're using in the DDL (because it works with liquibase, and the other one doesn't).